.github/workflows/build.yaml

```diff
@@ -41,6 +41,28 @@ jobs:
       date: ${{ inputs.date }}
       script: ci/build_cpp.sh
       sha: ${{ inputs.sha }}
+
+  rocky8-clib-standalone-build:
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/custom-job.yaml@main
+    strategy:
+      fail-fast: false
+      matrix:
+        cuda_version:
+          - &latest_cuda12 '12.9.1'
+          - &latest_cuda13 '13.0.2'
+    with:
+      build_type: ${{ inputs.build_type || 'branch' }}
+      branch: ${{ inputs.branch }}
+      arch: "amd64"
```

Comment:
Are we only supporting amd64 with the C library?

Reply:
I have on my todo to extend the matrix to arm + x86 but that isn't a hard requirement for 25.12.

If you know how to easily extend the matrix in a custom job I would happily integrate it into this PR.
When I was looking at other yaml files it seemed complicated ( dumping to file for some reason )

Reply:
Let's defer this until later. You can just add a `matrix` entry here like already exists for the CUDA keys and then use `${{ matrix.arch }}` in the `arch:` value but let's get one arch done before we expand.